src/actions/player/playerTrendsActions.js

```diff
@@ -63,9 +68,22 @@ export const getPlayerTrends = (playerId, options = {}, fieldName) => (dispatch)
           start_time: match.start_time,
           win,
         });
-
         return dataList;
       }, []);
+
+      // Apply mean-smoothing by taking differences of the previously computed integral
+      for (let i = 0; i < trends.length; i += 1) {
+        if (i < trends.length - chunkSize) {
+          trends[i].value = (trends[i + chunkSize].value - trends[i].value) / chunkSize;
```

Comment:
Gotcha. I've updated the loop to start from 20 and `data(i) = data(i) - data(i - chunkSize)` (using `i + chunSize` would invalidate the most recent 20. This way we only remove the first 20 from consideration as you asked)